packages/odyssey-react-mui/src/labs/TopNav.tsx

```diff
@@ -329,11 +329,11 @@ const AdditionalNavItemContainer = styled("div", {
 }));
 
 const TopNav = ({
-  SearchFieldComponent,
-  topNavLinkItems,
   AdditionalNavItemComponent,
-  settingsPageHref,
   helpPageHref,
+  SearchFieldComponent,
+  settingsPageHref,
+  topNavLinkItems,
```

Comment:
Out of personal curiosity: is this reordering to alphabetical order done automatically by a lint rule?

Reply:
Nope. It's a preference of ours.

I'd been meaning to add it to ESLint, but never got it prioritized. Auto-fixing those was always intended.